### PR TITLE
Add header file for  "hello world" example 

### PR DESCRIPTION
**Pull request by LiuYinCarl**
Add header file `stdio.h` to the example.Otherwise, writing the tests as documented will fail to compile.

### CODE REVIEW

**Comment by mwichmann** [comment]
Yes, agree. Most examples in the User Guide are encoded with special markup, so they're actually run as one of the doc build steps (and the output is captured for inclusion as well), and we see such omissions in those. However, this example is not... @bdbaddog, this looks good to me.

File: doc/user/simple.xml [hunk 1]
```diff
@@ -66,6 +66,8 @@
    </para>
 
    <programlisting>
+#include <stdio.h>
```
Comment (bdbaddog) [inline doc/user/simple.xml]: Change to include proper markup for < and >
```#include &lt;stdio.h&gt;```

**Comment by jcbrill** [comment]
FYI: also mentioned in https://github.com/SCons/scons/issues/4320#issuecomment-1460308313 concerning configure and lack of include for header resulting in linker error for some targets because there was no inline code generation (e.g., redefining printf to a different exported symbol) by the c preprocessor.

File: doc/user/simple.xml (hunk 1 above)
Comment (LiuYinCarl) [reply doc/user/simple.xml]: I will fix it soon, thanks.

I am learning scons for the use of [godot](https://github.com/godotengine/godot), and is learing scons's latest [User Guide](https://scons.org/doc/production/HTML/scons-user/index.html). Actually, the design of scons is very interesting to me, and I like the intuitive design. I will read the guide thoroughly. I might find additional typos that need to be fixed, so I reviewed the Github test failure report to learn how to run tests on my local computer and found the following commands.

```sh
# set up environment
python3 -m pip install --upgrade pip setuptools wheel
python3 -m pip install ninja psutil

# run document tests
python3 bin/docs-update-generated.py
python3 bin/docs-validate.py
python3 bin/docs-create-example-outputs.py

python3 runtest.py test/import.py test/ninja
```
I want to know if these commands is right for me, any reply is valuable, thanks.

**Comment by bdbaddog** [comment]
Thanks for the PR. Merging!

**Pull request merged**